decidim-core/app/views/devise/mailer/unlock_instructions.html.erb

```diff
@@ -0,0 +1,8 @@
+<% require 'devise/version' %>
```

Comment:
Debug?

Reply:
This is actually imported from `devise-i18n`, I don't know why they need it. I'll remove it!

Reply:
Ah, I see, that's weird!